### ISSUE TITLE
Add support for gradle continuous build

### ISSUE THREAD

**Issue by bodin** [bertramdev/asset-pipeline#37]
Currently when I try to use gradle's continuous build feature I receive the message 

> ./gradlew -t assetCompile
> ...
> Exiting continuous build as no executed tasks declared file system inputs.

http://gradle.org/feature-spotlight-continuous-build/

This would be nice to have in combination with incremental tasks

https://docs.gradle.org/current/userguide/custom_tasks.html#incremental_tasks

**Comment by davydotcom** [comment]
done in 2.6.x release

**Comment by bodin** [comment]
Just tried the 2.6.2 versions and I am receiving the same error about no declared inputs.  The file is processed when the task is run, but gradle exists once the first compile is done.  Let me know what else I can provide to help out.

{code}
classpath 'com.bertramlabs.plugins:less-asset-pipeline:2.6.2'
classpath 'com.bertramlabs.plugins:sass-asset-pipeline:2.6.2'
classpath 'com.bertramlabs.plugins:asset-pipeline-gradle:2.6.2'
{code}

 
{code}
assets {
	from 'src/main/webapp/assets'
	minifyJs = true
	minifyCss = true
	enableSourceMaps = true
	enableGzip = false
	enableDigests = false
	compileDir = 'src/main/webapp/public'
	includes = ['**/*.*']
}
{code}

**Comment by bodin** [comment]
I take back what I said - I t seems I can get this to work now.

I think what was causing the problem is the combination of
1) Using in a subproject
2) Defining the 'from' and 'compileDir' something other than the default

For now - the continuous integration is working. The other issues I had could probably be filed as a different issue, but I can not at this point determine enough information to provide a valid bug report.
